### DISCUSSION TITLE
Users not functioning properly

### DISCUSSION THREAD

**Discussion by Skylighty** [YetiForceCompany/YetiForceCRM#14967] [How to]
Hello, I've got contenerized yetiforce+mariadb+phpmyadmin (3 different containers, in one docker-compose). When i add the users they appear, and i can log in without a problem on their accounts, however when i put down the conteners by docker-compose and then turn it up again, logging onto previously created users accounts is not possible due to incorrect password, the only user that logs in is admin defined during installation wizard. The users are still visible in the yetiforce panel after logging onto admin and when i change their passwords it's possible to log in once again. The database is correct and it's config is as it should be. If the users should be in dav_users table of DB, then i see none there. Any fixes or any known cause of this error/bug?

The system is not registered properly yet if that's important.

Thanks in advance for the reply.

**Comment by mariuszkrzaczkowski** [comment]
run the logs, check server configuration,  it is best if you send us a docker for verification

**Reply by mariuszkrzaczkowski** [reply]
send information about the server because nothing is known about it,
![119500861-8d4fd580-bd68-11eb-84e0-9a381e3e002d](https://user-images.githubusercontent.com/10198654/125843918-96ed8fc2-25ff-4641-9c3b-da43727269f8.jpg)
